lib/graphql/metrics/analyzer.rb

```diff
@@ -51,20 +51,13 @@ def on_leave_field(node, parent, visitor)
           path: visitor.response_path,
         }
 
-        if GraphQL::Metrics.timings_capture_enabled?(query.context)
-          @static_field_metrics << static_metrics
-        else
-          field_extracted(static_metrics)
-        end
+        @static_field_metrics << static_metrics
       end
 
       def extract_fields(with_runtime_metrics: true)
-        return if query.context[SKIP_FIELD_AND_ARGUMENT_METRICS]
```

Comment:
`extract_fields` won't even be called now if `SKIP_FIELD_AND_ARGUMENT_METRICS` is already set anyway